repository/src/main/java/org/apache/atlas/repository/store/graph/v2/AtlasEntityStoreV2.java

```diff
@@ -2815,14 +2816,60 @@ public void unlinkBusinessPolicy(String policyGuid, Set<String> unlinkGuids) thr
         }
     }
 
-    private void handleBusinessPolicyMutation(List<AtlasVertex> vertices) throws AtlasBaseException {
+    @Override
+    @GraphTransaction
+    public void linkMeshEntityToAsset(String meshEntityGuid, Set<String> linkGuids) throws AtlasBaseException {
+        AtlasPerfMetrics.MetricRecorder metric = RequestContext.get().startMetricRecord("linkProductToAsset.GraphTransaction");
+
+        try {
+            List<AtlasVertex> vertices = this.entityGraphMapper.linkMeshEntityToAsset(meshEntityGuid, linkGuids);
+            if (CollectionUtils.isEmpty(vertices)) {
+                return;
+            }
+
+            LOG.info("linkMeshEntityToAsset: entityGuid={}, linkGuids={}", meshEntityGuid, linkGuids);
```

Comment:
Do not print whole linkGuids here, it will flood logs, just linkGuids.size() will be fine in case we want visibility